lib/qa/authorities/assign_fast/generic_authority.rb

```diff
@@ -0,0 +1,58 @@
+module Qa::Authorities
+  # A wrapper around the FAST api for use with questioning_authority
+  # API documentation:
+  # http://www.oclc.org/developer/develop/web-services/fast-api/assign-fast.en.html
+  class AssignFast::GenericAuthority < Base
+    attr_reader :subauthority
+    def initialize(subauthority)
+      @subauthority = subauthority
+    end
+
+    include WebServiceBase
+
+    # Search the FAST api
+    #
+    # @param [String] the query
+    # @return json results
+    def search q
+      begin
+        @raw_response = get_json(build_query_url(q))
+      rescue JSON::ParserError => error
+        # TODO: log the url and the error
+        return []
+      end
+      parse_authority_response
+    end
+
+    # Build a FAST API url
+    #
+    # @param [String] the query
+    # @return [String] the url
+    def build_query_url q
+      # certain chars not tolerated by the API
```

Comment:
It helps explain what the overall method does and I don't believe YARD is going to pick that line up when it creates the documentation, which it why I was advocating that it be placed above where the user could read it if there were consulting the docs.